python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -171,7 +171,7 @@ def _build_treelite_classifier(m, data, args, tmpdir):
         toolchain="gcc", libpath=model_path+"treelite.so",
         params={'parallel_comp': 40}, verbose=False
     )
-    return treelite.runtime.Predictor(model_path+"treelite.so", verbose=False)
+    return treelite_runtime.Predictor(model_path+"treelite.so", verbose=False)
```

Comment:
Minor style suggestion

```suggestion
    return treelite_runtime.Predictor(model_path + "treelite.so", verbose=False)
```

Reply:
Though maybe we should be using `os.path.join`?

Reply:
Using `os.path.join` now.